### DISCUSSION TITLE
在ElSelect中重新包裹ElOption做滚动使用下键不会跟随移动

### DISCUSSION THREAD

**Discussion by kcmeven** [element-plus/element-plus#12125] [Q&A]
![image](https://user-images.githubusercontent.com/48147837/226228822-d741d669-6ecc-43fe-9bd6-c42065bd04fb.png)

https://user-images.githubusercontent.com/48147837/226229179-88f0eca1-e2d6-4120-ba08-02b09be89b4a.mp4

该如何修改？

**Comment by gimjin** [comment]
使用 popper-class 属性设置宽高。

[demo](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtc2VsZWN0IHYtbW9kZWw9XCJ2YWx1ZVwiIHBvcHBlci1jbGFzcz1cIm15LWNsYXNzXCI+XG4gICAgPGVsLW9wdGlvblxuICAgICAgdi1mb3I9XCJpdGVtIGluIG9wdGlvbnNcIlxuICAgICAgOmtleT1cIml0ZW0udmFsdWVcIlxuICAgICAgOmxhYmVsPVwiaXRlbS5sYWJlbFwiXG4gICAgICA6dmFsdWU9XCJpdGVtLnZhbHVlXCJcbiAgICAvPlxuICA8L2VsLXNlbGVjdD5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgbGFuZz1cInRzXCIgc2V0dXA+XG5pbXBvcnQgeyByZWYgfSBmcm9tICd2dWUnXG5cbmNvbnN0IHZhbHVlID0gcmVmKCcnKVxuXG5jb25zdCBvcHRpb25zID0gW1xuICB7XG4gICAgdmFsdWU6ICdPcHRpb24xJyxcbiAgICBsYWJlbDogJ09wdGlvbjEnLFxuICB9LFxuICB7XG4gICAgdmFsdWU6ICdPcHRpb24yJyxcbiAgICBsYWJlbDogJ09wdGlvbjInLFxuICB9LFxuICB7XG4gICAgdmFsdWU6ICdPcHRpb24zJyxcbiAgICBsYWJlbDogJ09wdGlvbjMnLFxuICB9LFxuICB7XG4gICAgdmFsdWU6ICdPcHRpb240JyxcbiAgICBsYWJlbDogJ09wdGlvbjQnLFxuICB9LFxuICB7XG4gICAgdmFsdWU6ICdPcHRpb241JyxcbiAgICBsYWJlbDogJ09wdGlvbjUnLFxuICB9LFxuXVxuPC9zY3JpcHQ+XG5cbjxzdHlsZT5cbiAgLm15LWNsYXNzIHtcbiAgICBtaW4td2lkdGg6IDEwMHB4ICFpbXBvcnRhbnQ7XG4gICAgaGVpZ2h0OiAxMDBweDtcbiAgfVxuPC9zdHlsZT4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsIl9vIjp7fX0=)

**Reply by kcmeven** [reply]
[demo](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtc2VsZWN0IHYtbW9kZWw9XCJ2YWx1ZVwiIHBvcHBlci1jbGFzcz1cIlwiPlxuICAgIDxkaXYgY2xhc3M9XCJteS1jbGFzc1wiPlxuICAgICAgPGRpdj7moIfpopg8L2Rpdj5cbiAgICAgIDxkaXYgY2xhc3M9XCJjbHNcIj5cbiAgICAgICAgPGVsLW9wdGlvblxuICAgICAgICAgIHYtZm9yPVwiaXRlbSBpbiBvcHRpb25zXCJcbiAgICAgICAgICA6a2V5PVwiaXRlbS52YWx1ZVwiXG4gICAgICAgICAgOmxhYmVsPVwiaXRlbS5sYWJlbFwiXG4gICAgICAgICAgOnZhbHVlPVwiaXRlbS52YWx1ZVwiXG4gICAgICAgIC8+XG4gICAgICA8L2Rpdj5cbiAgICA8L2Rpdj5cbiAgPC9lbC1zZWxlY3Q+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgcmVmIH0gZnJvbSAndnVlJ1xuXG5jb25zdCB2YWx1ZSA9IHJlZignJylcblxuY29uc3Qgb3B0aW9ucyA9IFtcbiAge1xuICAgIHZhbHVlOiAnT3B0aW9uMScsXG4gICAgbGFiZWw6ICdPcHRpb24xJyxcbiAgfSxcbiAge1xuICAgIHZhbHVlOiAnT3B0aW9uMicsXG4gICAgbGFiZWw6ICdPcHRpb24yJyxcbiAgfSxcbiAge1xuICAgIHZhbHVlOiAnT3B0aW9uMycsXG4gICAgbGFiZWw6ICdPcHRpb24zJyxcbiAgfSxcbiAge1xuICAgIHZhbHVlOiAnT3B0aW9uNCcsXG4gICAgbGFiZWw6ICdPcHRpb240JyxcbiAgfSxcbiAge1xuICAgIHZhbHVlOiAnT3B0aW9uNScsXG4gICAgbGFiZWw6ICdPcHRpb241JyxcbiAgfSxcbl1cbjwvc2NyaXB0PlxuXG48c3R5bGU+XG4uY2xzIHtcbiAgaGVpZ2h0OiA4MHB4O1xuICBvdmVyZmxvdzogYXV0bztcbn1cbi5teS1jbGFzcyB7XG4gIGhlaWdodDogMTAwcHg7XG59XG48L3N0eWxlPlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJfbyI6e319)  这个样子

**Comment by gimjin** [comment]
看官方文档 https://element-plus.org/en-US/component/select.html#select-slots，slot tag 已经明确要求使用 Option Group / Option，div不在此列。
用 https://element-plus.org/en-US/component/select.html#grouping ，如果 group label 必须固定在顶部，你可以用 https://www.w3schools.com/cssref/tryit.php?filename=trycss_position_sticky 想办法吧。
